### DISCUSSION TITLE
cannot get my deployed nextjs site to reflect changes made on my sanity backend

### DISCUSSION THREAD

**Discussion by mhmmdrzaei** [vercel/community#4076] [Help]
### Summary

Hey all, 

I'm having a really hard time with this one -- i've build a sanity/next js site and it is working fine on my server however on putting it up on Vercel it will not update the content when changes are made to the sanity back end. 

I've tried literally every way i could think of -- adding
{
  cache: 'no-store'
  
}

or revalidate time period, have build a custom hook and also a hook through vercel. 

The hooks do work -- they trigger a rebuild however the content on the front end does not change. I've made sure to disable my cache on the page as well. I really am not sure what I'm doing wrong here. any feedback would be appreciated. I've used the new App routing to write my site as well. See attached github repo with site 

### Example

https://github.com/mhmmdrzaei/insearchof-new

### Steps to Reproduce

everything in repo. I've build a custom webhook -- it triggers and goes off but doesnt deploy the changes made on the sanity back end. What am i missing?

**Comment by ninjethics** [comment]
i was having this same problem, or similar. I found this helped me. 

just added this to the route and now my feed is updating with new content while deployed. Maybe it's the same for you? 

import { NextResponse } from 'next/server';

export const dynamic = 'force-dynamic';

https://github.com/orgs/vercel/discussions/3830
